backend/sample/schema.json

```diff
@@ -3,33 +3,22 @@
         {
             "id" : "__header__",
             "value" : [
-                {
-                    "id" : "stopwords:IDX",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                },
-                {
-                    "id" : "synonyms:IDX",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                }
+                {"type" : "string"}
             ]
         },
         {
-            "id" : "__comments__",
-            "value" : [
-                {
-                    "id" : "comment",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                }
-            ]
+            "id" : "stopwords:IDX",
+            "value" : {"type" : "string"},
+            "block_index" : {"type" : "integer"},
+            "block_subindex" : {"type" : "integer"}, 
+            "preceding_lines" : {"type" : "string"}

```

Comment:
Maybe it's a list of line ? I think it would be more appropriate (although this is not central).